### PR TITLE
tp-qemu: fix logical issue in watchdog.migration_when_wdt_timeout.pause

### PR DESCRIPTION
**Pull request by xutian**
Check vm paused during migration.

Signed-off-by: Xu Tian <xutian@redhat.com>

### CODE REVIEW

**Comment by xutian** [comment]
ID: 1234830

**Comment by CongSmile** [comment]
Acked-by: Cong Li <coli@redhat.com>

File: qemu/tests/cfg/watchdog.cfg [hunk 1]
```diff
@@ -41,8 +41,6 @@
                     watchdog_action = reset
                 - pause:
                     watchdog_action = pause
-                - poweroff:
-                    watchdog_action = poweroff
             variants:
```
Comment (suqinhuang) [inline qemu/tests/cfg/watchdog.cfg]: Hi xutian,
Why remove this scenerio.

Regards,
Suqin

Comment (xutian) [inline qemu/tests/cfg/watchdog.cfg]: after watchdog triggered, guest will be poweroff, if migration not finished, this case will always failed.

**Comment by suqinhuang** [comment]
Acked-by: Suqin Huang

**Pull request merged**